tests/foreman/cli/test_lifecycleenvironment.py

```diff
@@ -62,37 +62,24 @@ def test_bugzilla_1077333(self):
         @Assert: Can get info for lifecycle by its name
 
         """
-
-        payload = {
+        data = {
             'organization-id': self.org['id'],
             'name': gen_string('utf8', 15),
         }
 
-        new_obj = make_lifecycle_environment(payload)
-        self.assertIsNotNone(
-            new_obj, "Could not create lifecycle environment.")
+        result = LifecycleEnvironment.create(data)
```

Comment:
Will revert to the factory. When I saw this I miss thought that create just created the entity and not run the info, and this is not true. Thanks for catching this.